### ISSUE TITLE
Design The Look And Feel of the Member Survey

### ISSUE THREAD

**Issue by marcaaron** [Wobbly-App/wobbly-member-survey#5]
This survey could potentially be modeled after the design we currently have for [https://wobbly.app](https://wobbly.app). But if someone wants to help build on that chime in.

**Comment by david-ml** [comment]
This is about the max fanciness you get out of stock SurveyJS (according to one night of experience), but I bet we could liven it up!

![image](https://user-images.githubusercontent.com/7596288/46255471-f7eacf00-c462-11e8-8093-f178aa31040a.png)

**Comment by marcaaron** [comment]
Yes! This looks nice! I think we should run with this idea. Unless someone wants to build this from scratch (not me). Maybe we can think of an edge case where Survey JS doesn’t help us and have a challenge/contribution be to build a bespoke component for that. But I say this looks 🔥
